### PR TITLE
Update labels based on Zhiwu's branch

### PR DESCRIPTION
**Pull request by yinlinchen**

### CODE REVIEW

File: config/locales/sufia.en.yml [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+    account_name:       "PID"
     institution_name:   "Virginia Tech"
     product_name:       "VTechData"
     account_label:      "User"
-    deposit_agreement:  "VTechData's Deposit Agreement"
+    deposit_agreement:  "VTechData Deposit Agreement"
     document_language:  "en"
     upload_tooltip:     "Please accept Deposit Agreement before you can upload."
-    share_button:       "Share Your Work"
+    share_button:       "Share Your Dataset"
```
Comment (pmather) [inline config/locales/sufia.en.yml]: I thought the term "data set" was two words?

Comment (yinlinchen) [inline config/locales/sufia.en.yml]: @zhiwuxie  Please response this

Comment (zhiwuxie) [inline config/locales/sufia.en.yml]: one word is also ok, makes it easier to align with class name etc.

Comment (zhiwuxie) [inline config/locales/sufia.en.yml]: http://english.stackexchange.com/questions/2120/which-is-correct-dataset-or-data-set

**Comment by rotated8** [comment]
:+1: when squashed.

**Comment by yinlinchen** [comment]
@zhiwuxie next time please commit direct to the branch, do not directly edit file. Thx.

**Pull request closed without merging**